src/components/TabSelector/TabSelector.js

```diff
@@ -62,32 +62,43 @@ const getOpacity = (position, routesLength, tabIndex, active) => {
 
         return position.interpolate({
             inputRange,
-            outputRange: _.map(inputRange, (i) => (i === tabIndex ? activeValue : inactiveValue)),
+            outputRange: _.map(inputRange, (i) => (affectedTabs.includes(tabIndex) && i === tabIndex ? activeValue : inactiveValue)),
         });
     }
     return activeValue;
 };
 
-const getBackgroundColor = (position, routesLength, tabIndex) => {
+const getBackgroundColor = (position, routesLength, tabIndex, affectedTabs) => {
     if (routesLength > 1) {
         const inputRange = Array.from({length: routesLength}, (v, i) => i);
 
         return position.interpolate({
             inputRange,
-            outputRange: _.map(inputRange, (i) => (i === tabIndex ? themeColors.border : themeColors.appBG)),
+            outputRange: _.map(inputRange, (i) => (affectedTabs.includes(tabIndex) && i === tabIndex ? themeColors.border : themeColors.appBG)),
```

Comment:
Can you wrap the condition between parenthesis (to make sure it gets evaluated the right way)?
```suggestion
            outputRange: _.map(inputRange, (i) => ((affectedTabs.includes(tabIndex) && i === tabIndex) ? themeColors.border : themeColors.appBG)),
```

Reply:
The same for the other same condition, I just forgot to comment there.

Reply:
I already add them between parenthesis, but I run `npm run prettier` before commit the code and prettier remove them.

Reply:
Ohh prettier being picky, got it! Let's hope for the best then